lib/routes/3kns/index.ts

```diff
@@ -0,0 +1,122 @@
+import { Data, DataItem, Route } from '@/types';
+import got from '@/utils/got';
+import { getCurrentPath } from '@/utils/helpers';
+import { parseDate } from '@/utils/parse-date';
+import { art } from '@/utils/render';
+import { load } from 'cheerio';
+import { Context } from 'hono';
+import * as path from 'node:path';
+const __dirname = getCurrentPath(import.meta.url);
+
+export const route: Route = {
+    path: '/:category?/:language?/:tag?/:pubDate?/:collection?/:order?',
+    categories: ['game'],
+    example: '/3kns',
+    parameters: {
+        category: '游戏类型，见下表',
+        language: '游戏语言，见下表',
+        tag: '游戏标签，见下表',
+        pubDate: '发售时间，见下表',
+        collection: '游戏集合，见下表',
+        order: '排序，按高分排序:desc;按低分排序:asc',
+    },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: '3k-Switch游戏库',
+    maintainers: ['xzzpig'],
+    handler,
+    url: 'www.3kns.com/',
+    description: `游戏类型
+
+  | 不限  | 角色扮演 | 动作冒险 | 策略游戏 | 模拟经营 | 即时战略 | 格斗类 | 射击游戏 | 休闲益智 | 体育运动 | 街机格斗 | 无双类 | 其他游戏 | 赛车竞速 |
+  | --- | ---- | ---- | ---- | ---- | ---- | --- | ---- | ---- | ---- | ---- | --- | ---- | ---- |
+  | all | 1    | 2    | 3    | 4    | 5    | 6   | 7    | 8    | 9    | 10   | 11  | 12   | 13   |
```

Comment:
```suggestion
  | 不限 | 角色扮演 | 动作冒险 | 策略游戏 | 模拟经营 | 即时战略 | 格斗类 | 射击游戏 | 休闲益智 | 体育运动 | 街机格斗 | 无双类 | 其他游戏 | 赛车竞速 |
  | ---- | -------- | -------- | -------- | -------- | -------- | ------ | -------- | -------- | -------- | -------- | ------ | -------- | -------- |
  | all  | 1        | 2        | 3        | 4        | 5        | 6      | 7        | 8        | 9        | 10       | 11     | 12       | 13       |
```

Reply:
Are there any specific requirements or tools for formatting MarkDown table?

Reply:
A slight tweak of https://github.com/DIYgod/RSSHub/blob/stable/website/docs/.format/routeFormat.mjs